data_products/README.md

```diff
@@ -26,7 +25,9 @@ There is the full-sky simulation with background and all of the sources combined
 There is each of the sources individually with the background:
 - **Point_sources_10x_BG.tra.gz**: 4 point sources with 10x flux (Crab, Cyg X1, Cen A, Vela) and 1x Ling background
 - **Crab_BG_10x.tra.gz**: Crab with 10x flux and 1x Ling background
+- **CygX1_BG_10x.tra.gz**: Cyg X1 with 10x flux and 1x Ling background
 - **CenA_BG_10x.tra.gz**: Cen A with 10x flux and 1x Ling background
+- **Vela_BG_10x.tra.gz**: Vela with 10x flux and 1x Ling background
 - **GC511_10xFlux_and_Ling.inc1.id1.extracted.tra.gz**: 511 emission with 10x flux with Ling background
 - **DC1_Al26_10xFlux_and_Ling.inc1.id1.extracted.tra.gz**: Al26 emission with 10x flux with Ling background
```

Comment:
Maybe we should rename this to "Al26_10xFlux_and_Ling.inc1.id1.extracted.tra.gz", since the other files don't start with "DC1" prefix. Might not be worth the effort, though.